### ISSUE TITLE
Release 0.8.6 has DMG file with version 0.8.3

### ISSUE THREAD

**Issue by DutchPete** [pocketnetteam/pocketnet.gui#576]
On your release page for version 0.8.6 the `.dmg` file contains version 0.8.3.

**Comment by shpingalet007** [comment]
Thanks for your report @DutchPete.

I downloaded the DMG file to check your issue, but I'm not able to reproduce that, but without luck. Bastyon.app/Info.plist contains XML file with application data. There is a key `CFBundleShortVersionString` that for me is equal to 0.8.6.

How did you check the app version? Can you confirm that XML has 0.8.6 version too, as it is for me?

**Comment by DutchPete** [comment]
@shpingalet007: I downloaded the `.dmg` version and installed it.
After installation a pop-up appeared telling me there is a newer version! Huh?

I clicked on “later”, then checked in the bottom left corner of the app and noticed 0.8.3. So I went back to the download page to see if a newer version happened to have been uploaded just after my download. That was not the case because the latest upload was 3 days old.

So, I concluded that for the `.dmg` version the download link points to the wrong version.
